projects/core/src/state/reducers/transfer-state.reducer.ts

```diff
@@ -48,7 +48,10 @@ export function getServerTransferStateReducer(
     return function(state, action: any) {
       const newState = reducer(state, action);
       if (newState) {
-        const stateSlice = getStateSlice(Object.keys(keys), newState);
+        const stateSlice = getStateSlice(
+          filterKeysByType(keys, StateTransferType.TRANSFER_STATE),
```

Comment:
you should move `filterKeysByType` call out of reducer function for performance reasons